composer/loggers/console_logger.py

```diff
@@ -144,10 +160,13 @@ def _get_progress_string(self, state: State):
             training_progress = f'[{unit.name.lower()}={curr_duration}/{total}]'
         return training_progress
 
-    def log_to_console(self, data: Dict[str, Any], state: State, prefix: str = '') -> None:
+    def log_to_console(self, data: Dict[str, Any], state: State, prefix: str = '', is_train=True) -> None:
         # log to console
-        training_progress = self._get_progress_string(state)
-        log_str = f'{training_progress}:'
+        if is_train:
+            progress = self._get_progress_string(state)
+        else:
+            progress = self._get_eval_progress_string(state)
+        log_str = f'{progress}:'
```

Comment:
Can we not emit the `:` if there is nothing coming after that? It looks a bit strange in the logs.

Reply:
fixed